GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -179,7 +187,7 @@ public void ShowAuthor(bool show)
             this.DateColumn.Visible = show;
         }
 
-        [DefaultValue(FilterType.None)]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity"), DefaultValue(FilterType.None)]
```

Comment:
:exclamation: I'm not convinced this is correct. I would prefer to see it moved to its own pull request and evaluated separately.